variables.tf

```diff
@@ -1,9 +1,57 @@
-########################################################################################################################
+
+##############################################################################
 # Input Variables
-########################################################################################################################
+##############################################################################
+
+variable "region" {
+  description = "IBM Cloud region where all resources will be deployed"
+  type        = string
+}
+
+variable "resource_group_id" {
+  description = "ID of resource group to use when creating the VPC and PAG"
+  type        = string
+}
+
+variable "unique_name" {
+  description = "A unique identifier used as a prefix when naming resources that will be provisioned. Must begin with a letter."
+  type        = string
+}
+
+##############################################################################
+# Security and Compliance Center Workload Protection
+##############################################################################
+
+variable "scc_wp_service_plan" {
+  description = "IBM service pricing plan."
+  type        = string
+  default     = "free-trial"
+  validation {
+    error_message = "Plan for SCC Workload Protection instances can only be `free-trial` or `graduated-tier`."
+    condition = contains(
+      ["free-trial", "graduated-tier"],
+      var.scc_wp_service_plan
+    )
+  }
+}
+
+variable "scc_wp_tags" {
```

Comment:
I suggest you call it resource_tags to differentiate between them and access tags. Is access tags supported for this service? If so, you need to add a new variable to be able to add them

Reply:
As mentioned above, rename this to `resource_tags` - Did you check about access tags?